_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -199,6 +222,24 @@ The `aria-label` property is [prohibited][] for an element with a `generic` role
 <div aria-label="Bananas"></div>
 ```
 
+#### Failed Example 4
+
+The `aria-label` property is [prohibited][] for an element with a `paragraph` role.
+
+```html
+<div role="paragraph" aria-label="Bananas"></div>
+```
+
+#### Failed Example 5
+
+The `aria-valuemin`, `aria-valuemax` and `aria-valuenow` properties are not [supported][] for an element with a `separator` role that is not [focusable][].
```

Comment:
I agree that this ARIA says this is not allowed, but I feel there is a logical inconsistency in ARIA which makes that a questionable decision. Separators are allowed to be disabled. And when elements are disabled, ARIA says you're allowed to remove them from the focus order. But if you do that, you're suddenly no longer allowed to have these separator widget properties on it, including aria-disabled itself!

The problem is broader than just this role too. The spec says aria-disabled is applied to all focusable descendants. But then if you make an element not focusable because its disabled, according to the spec it then doesn't qualifies for inheriting that disabled state!

It's all a little too messy for my taste. I'd prefer we leave this example out. I for one wouldn't be comfortable failing this example.